src/hooks/api/gallery/useGetGalleryList.tsx

```diff
@@ -0,0 +1,170 @@
+import { useQuery, type UseQueryOptions } from '@tanstack/react-query';
+
+import { get } from '~/libs/api';
+import { type SurveyType, type TargetType } from '~/remotes/gallery';
+
+interface Request {
+  gallery_id?: string; //마지막으로 조회된 갤러리의 id를 입력
```

Comment:
```suggestion
interface Request {
  /**
   * 마지막으로 조회된 갤러리의 id를 입력
  */
  gallery_id?: string;
```

tsdoc 형태로 달아도 좋을 거 같아요 ~~

Reply:
수정했습니당!